main.js

```diff
@@ -0,0 +1,176 @@
+  },
+};
+
+const speakerImage1 = speakerObject.one.image;
+const speakerName1 = speakerObject.one.name;
+const speakerQuali1 = speakerObject.one.qualifications;
+const speakerInfo1 = speakerObject.one.info;
+
+if (window.location.pathname === '/Agro-Summit/index.html') {
+  speakName1.innerHTML = speakerName1;
+  speakImg1.src = speakerImage1;
+  speakQuali1.innerHTML = speakerQuali1;
+  speakInfo1.innerHTML = speakerInfo1;
+}
+
+const speakerImage2 = speakerObject.two.image;
+const speakerName2 = speakerObject.two.name;
+const speakerQuali2 = speakerObject.two.qualifications;
+const speakerInfo2 = speakerObject.two.info;
+
+if (window.location.pathname === '/Agro-Summit/index.html') {
+  speakName2.innerHTML = speakerName2;
+  speakImg2.src = speakerImage2;
+  speakQuali2.innerHTML = speakerQuali2;
+  speakInfo2.innerHTML = speakerInfo2;
+}
+
+const speakerImage3 = speakerObject.three.image;
+const speakerName3 = speakerObject.three.name;
+const speakerQuali3 = speakerObject.three.qualifications;
+const speakerInfo3 = speakerObject.three.info;
+
+if (window.location.pathname === '/Agro-Summit/index.html') {
+  speakName3.innerHTML = speakerName3;
+  speakImg3.src = speakerImage3;
+  speakQuali3.innerHTML = speakerQuali3;
+  speakInfo3.innerHTML = speakerInfo3;
+}
+
+const speakerImage4 = speakerObject.four.image;
+const speakerName4 = speakerObject.four.name;
+const speakerQuali4 = speakerObject.four.qualifications;
+const speakerInfo4 = speakerObject.four.info;
+
+if (window.location.pathname === '/Agro-Summit/index.html') {
+  speakName4.innerHTML = speakerName4;
+  speakImg4.src = speakerImage4;
+  speakQuali4.innerHTML = speakerQuali4;
+  speakInfo4.innerHTML = speakerInfo4;
+}
+
+const speakerImage5 = speakerObject.five.image;
+const speakerName5 = speakerObject.five.name;
+const speakerQuali5 = speakerObject.five.qualifications;
+const speakerInfo5 = speakerObject.five.info;
+
+if (window.location.pathname === '/Agro-Summit/index.html') {
+  speakName5.innerHTML = speakerName5;
+  speakImg5.src = speakerImage5;
+  speakQuali5.innerHTML = speakerQuali5;
+  speakInfo5.innerHTML = speakerInfo5;
+}
+
+const speakerImage6 = speakerObject.six.image;
+const speakerName6 = speakerObject.six.name;
+const speakerQuali6 = speakerObject.six.qualifications;
+const speakerInfo6 = speakerObject.six.info;
+
+if (window.location.pathname === '/Agro-Summit/index.html') {
+  speakName6.innerHTML = speakerName6;
+  speakImg6.src = speakerImage6;
+  speakQuali6.innerHTML = speakerQuali6;
+  speakInfo6.innerHTML = speakerInfo6;
+}
+
+const showSpeakers = document.querySelector('#show-more');
+const otherSpeakers = document.querySelector('.play-part');
+
+function displayMore(event) {
+  otherSpeakers.classList.toggle('ham-gone');
+  event.preventDefault();
+}
+
+if (window.location.pathname === '/Agro-Summit/index.html') {
+  showSpeakers.addEventListener('click', displayMore);
+}
```

Comment:
- [OPTIONAL] I would like to suggest you the more button, it would be better when the user clicks on the more button and it shows more speakers the button changed to less ` LESS ⬆️ ` instead of ` MORE ⬇️ ` 😉 😉